pallets/attestation/src/lib.rs

```diff
@@ -152,6 +160,14 @@ pub mod pallet {
 		BoundedVec<ClaimHashOf<T>, <T as Config>::MaxDelegatedAttestations>,
 	>;
 
+	/// Attestations stored on chain.
+	///
+	/// It maps from a claim hash to the full attestation.
```

Comment:
M: Fix document. This was copy-pased from `Attestations` 😁

Reply:
fixed c415bd9